dspace-server-webapp/src/main/java/org/dspace/app/rest/signposting/relation/ItemAuthorProcessor.java

```diff
@@ -0,0 +1,109 @@
+import org.dspace.util.FrontendUrlService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * An implementation of {@link ItemSignPostingProcessor} for the author relation.
+ *
+ * @author Luigi Andrea Pascarelli (luigiandrea.pascarelli at 4science.com)
+ * @author Francesco Pio Scognamiglio (francescopio.scognamiglio at 4science.com)
+ */
+public class ItemAuthorProcessor extends ASignPostingProcessor
+    implements ItemSignPostingProcessor {
+
+    /** log4j category */
+    private static Logger log = Logger.getLogger(ItemAuthorProcessor.class);
+
+    @Autowired
+    private ItemService itemService;
+
+    @Autowired
+    private FrontendUrlService frontendUrlService;
+
+    private String retrievedExternally;
+
+    public String getRetrievedExternally() {
+        return retrievedExternally;
+    }
+
+    public void setRetrievedExternally(String retrievedExternally) {
+        this.retrievedExternally = retrievedExternally;
+    }
+
+    public ItemAuthorProcessor() {
+        setRelation("author");
+    }
+
+    @Override
+    public void buildRelation(Context context, HttpServletRequest request,
+                              Item item, List<Linkset> linksets, Linkset primaryLinkset) {
+        try {
+            List<MetadataValue> authors = itemService
+                    .getMetadata(item, MetadataSchemaEnum.DC.getName(), "contributor", ANY, ANY);
+            for (MetadataValue author : authors) {
+                if (author.getAuthority() != null) {
+                    String authorUuid = author.getAuthority();
+                    Item authorItem = itemService.find(context, UUID.fromString(authorUuid));
+                    String authorOrcid = itemService
+                            .getMetadataFirstValue(authorItem, PERSON.getName(), "identifier", "orcid", ANY);
+                    if (StringUtils.isNotBlank(authorOrcid)) {
+                        String href = frontendUrlService.generateUrl(authorItem);
+                        primaryLinkset.getAuthor().add(new Relation(href, authorOrcid));
+                    }
+                }
+            }
+        } catch (Exception e) {
+            log.error("Problem to add signposting pattern", e);
+        }
+    }
+
+    @Override
+    public void buildLset(Context context, HttpServletRequest request,
+                          Item item, List<Lset> lsets) {
+        try {
+            List<MetadataValue> authors = itemService
+                    .getMetadata(item, MetadataSchemaEnum.DC.getName(), "contributor", ANY, ANY);
+            for (MetadataValue author : authors) {
+                if (author.getAuthority() != null) {
+                    String authorUuid = author.getAuthority();
+                    Item authorItem = itemService.find(context, UUID.fromString(authorUuid));
+                    String authorOrcid = itemService
+                            .getMetadataFirstValue(authorItem, PERSON.getName(), "identifier", "orcid", ANY);
```

Comment:
this require rework. The above implementation is ok for DSpace-CRIS but not for DSpace. In DSpace we should use the relation to retrieve authors and from there take the `person.identifier.orcid`. I don't think that we should provide support for the ORCID authority at this time, better to invest on the future / most powerful way to manage the ORCID integration. Indeed the ORCID Authority is not used also in other interoperability layer such as the OAI-PMH where only relation are taken in consideration, so this would make also the application behavior more consistent